Automattic/wp-calypso
# Gutenberg: Disable NUX tour tooltips

#### Changes proposed in this Pull Request

Disable "Welcome to the Wonderful World of Blocks!" tooltip / guided tour.

#### Testing instructions

* Visit the block editor on a new site (one on which you have not previously opened it) or on a browser after clearing localStorage cache
* The "Welcome to the Wonderful World of Blocks!" tooltip should not appear
* Will fix #32472 

Please see: PCYsg-l4k-p2 for additional testing steps
